app/src/main/java/eu/kanade/tachiyomi/data/source/online/OnlineSource.kt

```diff
@@ -54,6 +54,11 @@ abstract class OnlineSource(context: Context) : Source {
     abstract val lang: Language
 
     /**
+     * If the Source has Support for Latest Updates
+     */
+    abstract val supportsLatest : Boolean
```

Comment:
Initially I changed this thinking it would cause less overhead but I later realized at compile time its likely just a get() so its the same